man/shards.1

```diff
@@ -1,50 +1,48 @@
-.TH SHARDS "1" "September 2015" "shards 0.8.1" "User Commands"
+.TH SHARDS "1" "September 2018" "shards 0.8.1" "User Commands"
 .SH NAME
 shards \- dependency manager for the Crystal Language
 .SH SYNOPSYS
 .B
 shards
-[\fIOPTION\fR]... [\fICOMMAND\fR]...
+[\fI<options>\fR...] <\fIcommand\fR>
```

Comment:
Unlike `git` we don't have to specify a command since `install` is the default command. It should be `[command]` not `<command>`.

Thought, I'd be fine to *require* a command to be specified, now that the `crystal deps` shortcut has been removed.

Reply:
Right, that was really just a typo and is correct in the embedded help.